src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/Snapshot/Dependency.cs

```diff
@@ -266,7 +266,9 @@ private static string GetAlias(IDependency dependency)
 
         private static string Normalize(string id)
         {
-            return id.Replace('/', '\\');
+            return id
+                .Replace('/', '\\')
+                .Replace("..", "__");
```

Comment:
I have not seen a regression in Analyzers. I believe Analyzers use an absolute path so it would not be impacted by this.